example/angular1/pubspec.yaml

```diff
@@ -1,10 +1,9 @@
 name: scissors_angular1_example
```

Comment:
Something is wrong in our understanding of rebase... can you please fix this new rebase hell?